resolwe_bio/processes/import_data/hmmer_database.yml

```diff
@@ -7,8 +7,10 @@
   name: Upload HMMER database
   requirements:
     expression-engine: jinja
+    resources:
+      network: true
```

Comment:
Check if the network resource is really needed as you are not using re-import in this process. Either support re-import or remove the network requirement.